pkgs/applications/science/logic/isabelle/default.nix

```
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchurl, coreutils, nettools, java, scala, polyml, z3, veriT, vampire, eprover-ho, naproche, rlwrap, perl, makeDesktopItem, isabelle-components, isabelle, symlinkJoin, fetchhg }:
+{ lib, stdenv, fetchurl, coreutils, nettools, java, scala_3, polyml, z3, veriT, vampire, eprover-ho, naproche, rlwrap, perl, makeDesktopItem, isabelle-components, isabelle, symlinkJoin, fetchhg, srcRev ? null, srcSha256 ? null, srcUrl ? "https://isabelle.sketis.net/repos/isabelle" }:
```

```suggestion
{ lib, stdenv, fetchurl, coreutils, nettools, java, scala_3, polyml, z3, veriT, vampire
, eprover-ho, naproche, rlwrap, perl, makeDesktopItem, isabelle-components, isabelle
, symlinkJoin, fetchhg }:
```
That line is a bit long. Also those by default null arguments are not good style.